dali/test/python/operator/test_one_hot.py

```diff
@@ -1,4 +1,4 @@
-# Copyright (c) 2017-2020, NVIDIA CORPORATION. All rights reserved.
+# Copyright (c) 2017-2022, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
```

Comment:
Yeah, but nothing changed in this file 😅

Reply:
I was debugging with this file and it got smashed by the githook.